src/Event/AdminEventExtension.php

```diff
@@ -21,6 +21,7 @@
 use Sonata\AdminBundle\Form\FormMapper;
 use Sonata\AdminBundle\Show\ShowMapper;
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface as ContractsEventDispatcherInterface;
```

Comment:
Let's try to make the `NEXT_MAJOR` minimal. Let's start by aliasing the other interface.